app/views/results_entities.php

```diff
@@ -23,12 +23,18 @@
         $path_locale2 = VersionControl::hgPath($locale, $check['repo'], $entity);
     }
 
-    $target_string2 = '';
+    // Escape strings for HTML Display
+    $bz_target_string = $target_string = isset($tmx_target[$entity])
+                                            ? Utils::secureText($tmx_target[$entity])
+                                            : '';
+    $bz_target_string2 = $target_string2 = isset($tmx_target2[$entity])
```

Comment:
You should put this definition in the following if cycle, we don't need it for the normal view, only for 3locales.